pkg/services/notifications/smtp.go

```diff
@@ -93,6 +94,7 @@ func (sc *SmtpClient) buildEmail(ctx context.Context, msg *Message) *gomail.Mess
 	m.SetHeader("From", msg.From)
 	m.SetHeader("To", msg.To...)
 	m.SetHeader("Subject", msg.Subject)
+	m.SetHeader("Message-ID", uuid.New().String())
```

Comment:
Easier to just use `uuid.NewString`:

```suggestion
	m.SetHeader("Message-ID", uuid.NewString())
```

Reply:
Actually, on reading the RFC, I don't think this is the right format.

The RFC specifies a form like:

```
   msg-id          =   [CFWS] "<" id-left "@" id-right ">" [CFWS]
```

So, I think this should be something like

```suggestion
	m.SetHeader("Message-ID", fmt.Sprintf("<%s@%s>", uuid.NewString(), domain))
```

(where `domain` needs to be extracted from `msg.From`)

Reply:
Thanks for the comments! I've processed them both.